codeception.dist.yml

```diff
@@ -24,10 +24,10 @@ modules:
       composer_magento_username: "%REPO_USERNAME%"
       composer_magento_password: "%REPO_PASSWORD%"
       composer_github_token: "%GITHUB_TOKEN%"
-      printOutput: false
+      printOutput: true
```

Comment:
Is it intended to change?

Reply:
I changes it for testing.
Need to change back in the next PR: https://github.com/magento/ece-tools/pull/766

Reply:
Ok

Reply:
Also, I'm going to add ability to change it via env variables.

> Can you please test that if \Magento\Framework\Cache\Backend\RemoteSynchronizedCache and Redis_Slave_Enabled is configured the slave configuration go to proper section (remote_backend_options)?

I have checked it on pro env.
Looks like it works fine.